tests/Doctrine/Tests/ORM/Tools/EntityGeneratorTest.php

```diff
@@ -224,6 +224,26 @@ public function newInstance($metadata)
         return new $metadata->name;
     }
 
+    /**
+     * @group GH-6314
+     */
+    public function testEmbeddedEntityWithColumnPrefix()
+    {
+        $testMetadata = $this->generateTestEmbeddableFixture();
+        $isbnMetadata = $this->generateIsbnEmbeddableFixture(['testEmbedded' => $testMetadata], 'prefix');
+        $isbnEntity = $this->newInstance($isbnMetadata);
+        $refClass = new \ReflectionClass($isbnEntity);
+
+        self::assertTrue($refClass->hasProperty('testEmbedded'));
+
+
+        $docComment = $refClass->getProperty('testEmbedded')->getDocComment();
+
+        $needle = sprintf('@Embedded(class="%s", columnPrefix=\'%s\')', $testMetadata->name,'prefix');
```

Comment:
Fairly sure that we don't use single quotes on any annotations (see `columnPrefix`)